packages/react/README.md

```diff
@@ -0,0 +1,25 @@
+# JSONForms - More Forms. Less Code
+### Complex Forms in the blink of an eye
+
+JSONForms uses HTML custom elements and eliminates the need to write HTML templates in order to create forms by leveraging the capabilities of JSON and JSON schema.
+
+# React Package
+This is the JSONForms react package which provides the necessary bindings for React.
+
+You can combine the react package with any renderer you want, for example the [Material Renderers](https://www.npmjs.com/package/@jsonforms/material-renderers) or the [Vanilla Renderers](https://www.npmjs.com/package/@jsonforms/vanilla-renderers).
```

Comment:
Either write 'any renderer set' or change to 'Material Renderer[s]'
Maybe we should also be a bit more precise about the wording here since any react-based renderer set is meant here.